docs/UserGuide.md

```diff
@@ -319,6 +319,7 @@ Application: Used to find a certain task that may be further down the list.
 Format: `find KEYWORD`
 
 * Searches through ModuleBook3.5 for tasks whose names contain `KEYWORD`.
+* You may contain multiple keywords. ModuleBook3.5 will list down all tasks that contains any of the given keyword.
```

Comment:
I think "You may **contain** multiple keywords." should be changed to "You may **key in** multiple keywords"

Reply:
nice suggestion, fixed